scanner/src/main/kotlin/scanners/Askalono.kt

```diff
@@ -105,7 +99,9 @@ class Askalono internal constructor(
         return unpackDir
     }
 
-    override fun scanPathInternal(path: File): ScanSummary {
+    override fun filterSecretOptions(options: Options): Options = options
```

Comment:
I was about to suggest this should become a default implementation, but then saw you do this later. Maybe move the commit with the default implementation to earlier, to reduce the diff here (and for the other scanners)?

Reply:
This wasn't possible earlier, because then there would have been two default implementations, one from `Scanner` and one from `ScannerWrapper` so all implementing classes would have had to re-implement anyway. It was easier to just wait until no class inherits `Scanner` anymore.